app/Http/Controllers/TopController.php

```diff
@@ -7,8 +7,8 @@
 class TopController extends Controller
 
 {
-    public function top(Request $request,$id){
+    public function top( ){
```

Comment:
概ね良いのですが、半角スペースは無しでOKです！
```suggestion
    public function top(){
```